### ISSUE TITLE
RuntimeError: Found 0 images in subfolders of: data/ClothingAttributeDataset/train/

### ISSUE THREAD

**Issue by pisicutza21** [sampathweb/insight-clothing-attributes#2]
Hy! I'm trying to learn machine learning, and from this how to extract from a photo different attributes. I looked through your project, trying to figure out how you do it, but when i'm running it i got this message error RuntimeError: Found 0 images in subfolders of: data/ClothingAttributeDataset/train/, even if i created the folder data/ClothingAttributeDataset/train/, and put  images into. can you please tell me, what i'm doing wrong, or what i missed? Thank you :).

**Comment by sahnefreak** [comment]
Hey! I have the same issue. Do you have any solution yet? Best regards! :)

**Comment by tlsgb456** [comment]
@pisicutza21 @sahnefreak 
I think we should download 'Clothing Attributes Dataset' on this webpage. 
https://purl.stanford.edu/tb980qz1002

**Comment by pisicutza21** [comment]
@tlsgb456 @sahnefreak  Did it worked  with the dataset from that webpage or you did some changes? Can you please tell me how much ram do you have? I'm asking because i  have this error "RuntimeError: $ Torch: not enough memory: you tried to allocate 7GB. Buy new RAM! at /opt/conda/conda-bld/pytorch_1523244252089/work/torch/lib/TH/THGeneral.c:253", i don't know how to fixed it. Thnx :)

**Comment by tlsgb456** [comment]
@pisicutza21 If you lack the VRAM, you can use small batch size such as 16, 8, 4, 2, 1 because that default batch size is 32.

**Comment by pisicutza21** [comment]
@tlsgb456 you are right, thanks for the answers. I changed batch size to 4, i have 1,9 Gb ram free. It's been four hours since i compiled build_models, and the last thing shown is valid Epoch 0/49, it looks like it is blocked. how much time did you need to train the models for the 50 epochs?

**Comment by tlsgb456** [comment]
@pisicutza21 I use TITAN X,  so I can use 32 batch size. 50 epoch train take ... um... approximately 30 min? If you using smaller batch size, you may take more time than my case.

**Comment by sagarnildass** [comment]
The documentation is poor here. Here are the steps to get a good result:

1. First download the ClothingAttributeDatasetand put it in the data folder.

2. In the ml_src folder, run the data-prep.ipynb. Don't spilt train test by hand.

3. In the ml_src/weights folder, create a folder called vgg16-fcn-266-2

3. Run python ml_src/build_models.py from outside the ml_src folder.
